packages/sequelize-model/test/migrations/20191105141904_createMinistryTable.js

```diff
@@ -0,0 +1,57 @@
+module.exports = {
+  up: (queryInterface, Sequelize) => {
+    return queryInterface.createTable(
```

Comment:
if I understand right, this file is now redundant, as we have the migrations defined on root package level